package/utils/helpers.js

```diff
@@ -37,11 +38,24 @@ const canProcess = (rule, fn) => {
   return null
 }
 
+const asBoolean = (str) => {
```

Comment:
thanks @masato-hi Could you please use the above function `isBoolean` (line 2)?

Reply:
Thanks for review.
Cannot pass `"false"` in the `isBoolean` function.